site/en/blog/extension-news-october-2023/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's happening in Chrome Extensions?
+description: >
+ An overview of the changes so far this year in Chrome Extensions, plus exciting upcoming extension features developers can look forward to.
```

Comment:
>so far this year

Do you mean this quarter? If you say 'recent changes' you won't need to remember to update it every time.

Reply:
Great suggestion 💯 !!! Let's go with recent changed 👍